client/src/Components/CommonComponent/Table/style.css

```diff
@@ -1,14 +1,50 @@
+input[type="text"]::placeholder {
+  text-align: right;
+}
+input[type="text"]:-ms-input-placeholder {
+  text-align: right;
+}
```

Comment:
@Anies12 These styles are global, so if you want to ensure that they only apply to the Table component you'll need to make them a little more specific, for example `.table-container input[type="text"]::placeholder {`

Reply:
well, I'm actually doing that is many other places, but I wanted to ensure that every input field in the application is aligned to the right, as we are building an Arabic-language application

Reply:
Ah got it.  In that case, maybe these styles belong in a more "global" location, like App.css or something.  But it's not super important 🤷‍♂️

Reply:
sure, it would be better to put it in App.css